server/db/api/composite/SceneConstellation.ts

```diff
@@ -211,6 +211,7 @@ export class SceneConstellation {
 
             let nonModelAssets: SvxNonModelAsset[] | null = null;
             if (svx.SvxExtraction.nonModelAssets) {
+                const AV = await AssetVersion.fetch(idAssetVersion);

```

Comment:
Error-checking is wanted here.  AV should not be null ... but will be in the event of an error.  In that case, we should, at a minimum, LOG an error.  What error handling should occur next -- we likely should return null to indicate the error to the caller.

All code needs error handling.  Server-side code really needs it.